themes/default/layouts/page/automation.html

```diff
@@ -3,193 +3,422 @@
-                                    <a href="{{ relref . "/case-studies/snowflake" }}" class="link">
-                                        {{ partial "customer-logo.html" (dict "logo" "snowflake") }}
-                                    </a>
+        <div class="container mx-auto mb-16">
+            <h2 class="my-12 text-center">Use Cases</h2>
+
+            <div class="automation-scenarios-carousel-items">
+                <span id="self-service" class="hidden"></span>
+                <span id="saas" class="hidden"></span>
+                <span id="workflow" class="hidden"></span>
+                <span id="tooling" class="hidden"></span>
+
+                <ul id="automation-scenarios-carousel-headers" class="automation-scenarios-carousel-headers">
+                    <li class="" data-product-carousel-type="self-service">
+                        <h5 class="my-0 text-black xl:block hidden"><a href="#self-service" class="text-black">Self-service infrastructure</a></h5>
+                        <h6 class="my-0 text-black xl:hidden"><a href="#self-service" class="text-black">Self-service infrastructure</a></h6>
+                    </li>
+                    <li class="" data-product-carousel-type="saas">
+                        <h5 class="my-0 xl:block hidden"><a href="#saas" class="text-gray-500">Single-tenant SaaS instances</a></h5>
+                        <h6 class="my-0 xl:hidden"><a href="#saas" class="text-gray-500">Single-tenant SaaS instances</a></h6>
+                    </li>
+                    <li class="" data-product-carousel-type="workflow">
+                        <h5 class="my-0 xl:block hidden"><a href="#workflow" class="text-gray-500">Infrastructure Workflow Orchestration</a></h5>
+                        <h6 class="my-0 xl:hidden"><a href="#workflow" class="text-gray-500">Infrastructure Workflow Orchestration</a></h6>
+                    </li>
+                    <li class="" data-product-carousel-type="tooling">
+                        <h5 class="my-0 xl:block hidden"><a href="#tooling" class="text-gray-500">Custom Infrastructure Tooling</a></h5>
+                        <h6 class="my-0 xl:hidden"><a href="#tooling" class="text-gray-500">Custom Infrastructure Tooling</a></h6>
+                    </li>
+                </ul>
+
+                <ul class="automation-scenarios-carousel-items-list">
+                    <li class="w-full pb-3" data-product-carousel-type="self-service">
+                        <h3 class="lg:hidden text-center">Self-service infrastructure</h3>
+                        <div class="flex flex-wrap">
+                            <div class="w-full lg:w-2/3 py-3 px-6">
+                                <div class="card p-3 px-6 relative bg-white">
+                                    <p>
+                                        Build custom infrastructure tools, developer portals, and platforms that
```

Comment:
Build developer portals and self-serve platforms that make it easier for your development teams to use cloud infrastructure. Infrastructure and security teams can scale their best practices by creating declarative infrastructure and exposing it behind a REST, gRPC, or Custom Resource API that developers and operators can easily consume, without needing to understand every underlying resource.

Reply:
👍